server/api/api/endpoints/model_monitoring.py

```diff
@@ -35,6 +35,29 @@ class _CommonParams:
     project: str
     auth_info: mlrun.common.schemas.AuthInfo
     db_session: Session
+    model_monitoring_access_key: Optional[str] = None
+
+    def __post_init__(self) -> None:
+        if not mlrun.mlconf.is_ce_mode():
+            # Generate V3IO Access Key
```

Comment:
Probably "generate" is the wrong word. Seems like it just gets it from the project secrets.

Reply:
4c70829